sql/core/benchmarks/CollationNonASCIIBenchmark-jdk21-results.txt

```diff
@@ -2,26 +2,53 @@ OpenJDK 64-Bit Server VM 21.0.2+13-LTS on Linux 6.5.0-1017-azure
+UTF8_BINARY                                           483            489          11          0.1       12084.1      38.9X
+UNICODE_CI                                          15285          15287           3          0.0      382114.7       1.2X
 
 OpenJDK 64-Bit Server VM 21.0.2+13-LTS on Linux 6.5.0-1017-azure
 AMD EPYC 7763 64-Core Processor
 collation unit benchmarks - compareFunction:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ---------------------------------------------------------------------------------------------------------------------------
-UTF8_BINARY_LCASE                                    18377          18399          31          0.0      459430.5       1.0X
-UNICODE                                              14238          14240           3          0.0      355957.4       1.3X
-UTF8_BINARY                                            975            976           1          0.0       24371.3      18.9X
-UNICODE_CI                                           13819          13826          10          0.0      345482.6       1.3X
+UTF8_BINARY_LCASE                                    18822          18831          13          0.0      470547.2       1.0X
+UNICODE                                              14491          14505          21          0.0      362268.8       1.3X
+UTF8_BINARY                                            980            981           1          0.0       24487.8      19.2X
+UNICODE_CI                                           14494          14524          43          0.0      362341.6       1.3X
 
 OpenJDK 64-Bit Server VM 21.0.2+13-LTS on Linux 6.5.0-1017-azure
 AMD EPYC 7763 64-Core Processor
 collation unit benchmarks - hashFunction:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-UTF8_BINARY_LCASE                                  9183           9230          67          0.0      229564.0       1.0X
-UNICODE                                           38937          38952          22          0.0      973421.3       0.2X
-UTF8_BINARY                                        1376           1376           0          0.0       34397.5       6.7X
-UNICODE_CI                                        32881          32882           1          0.0      822027.4       0.3X
+UTF8_BINARY_LCASE                                 10768          10780          17          0.0      269199.8       1.0X
+UNICODE                                           42046          42062          23          0.0     1051138.9       0.3X
+UTF8_BINARY                                        1980           1984           5          0.0       49506.4       5.4X
+UNICODE_CI                                        33206          33212           7          0.0      830160.4       0.3X
```

Comment:
I see that the benchmark runs on previous functions too, but we shouldn't make any unnecessary modifications in this PR (that are not related to the changes brought by this PR)

so I would advise keep the previous results as they were, and only append yours below